sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -213,11 +298,61 @@ export interface SetReadOnlyOptions extends HttpOnlyIfUnchangedField, OperationO
 export interface SetReadOnlyResponse extends ConfigurationSetting, SyncTokenHeaderField, HttpResponseField<SyncTokenHeaderField> {
 }
 
+// @public (undocumented)
+export interface Snapshot {
+    compositionType?: CompositionType;
+    readonly createdOn?: Date;
+    readonly etag?: string;
+    readonly expiresOn?: Date;
+    filters: ConfigurationSettingsFilter[];
+    readonly itemCount?: number;
+    readonly name: string;
+    retentionPeriod?: number;
+    readonly size?: number;
+    readonly status?: SnapshotStatus;
+    readonly statusCode?: number;
+    tags?: {
+        [propertyName: string]: string;
+    };
+}
```

Comment:
I guess I don't fully follow why some of the fields in here are readonly and others aren't.